snapshot_dbg_extension/src/adapter.ts

```diff
@@ -64,6 +64,7 @@ export class SnapshotDebuggerSession extends DebugSession {
     private setVariableType: boolean = false;
     private userPreferences: UserPreferences;
     private isDeferredInitializationDone = false;
+    private activeInitializePathCount = 0;
```

Comment:
It's worth adding a comment in here; the intention behind this variable is not obvious to me.